src/Command/ReconcileCommand.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+
+namespace Webgriffe\SyliusAkeneoPlugin\Command;
+
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Command\LockableTrait;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webgriffe\SyliusAkeneoPlugin\ImporterInterface;
+use Webgriffe\SyliusAkeneoPlugin\ReconcilerRegistryInterface;
+use Webgriffe\SyliusAkeneoPlugin\ReconcilerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor $lock
+ */
+final class ReconcileCommand extends Command
+{
+    use LockableTrait;
+
+    public const SUCCESS = 0;
+
+    public const FAILURE = 1;
+
+    private const IMPORTER_OPTION_NAME = 'importer';
```

Comment:
Even if the `ReconcilerInterface` will be implemented by the importer I'd use `reconciler` here as option name:
```suggestion
    private const RECONCILER_OPTION_NAME = 'reconciler';
```